packages/relay/src/lib/mirrorNode.ts

```diff
@@ -134,4 +134,62 @@ export class MirrorNode {
     const block = this.store.get(hash);
     return block === undefined ? null : block;
   }
+
+  public async getContractResult(hash: string): Promise<any> {
```

Comment:
I don't believe this is called anywhere, and it should be as it should be phased out by the actual implementation you've provided in `mirrorNodeClient`